### DISCUSSION TITLE
"Previous err" line since commit 416c1ee

### DISCUSSION THREAD

**Discussion by Kinsiinoo** [qdm12/gluetun#1475] [Q&A] [answered]
Hi,
Is it normal that a bunch of this line appear since commit 416c1ee?
`==== previous err:  <nil>`

Every few seconds a new one appears in the log when I look at the docker log.

Thanks in advance!

**Comment by Kinsiinoo** [comment]
I found it in commit fe97e28, but it never ends, it always print a similar new line.

**Comment by qdm12** [comment, marked as answer]
Fixed it, thanks for mentioning it 😉 Forgot to remove the debug line!
